ethpm_types/contract_type.py

```diff
@@ -204,6 +205,44 @@ def extract_sourcemap_item(expanded_row, item_idx, previous_val=None):
             yield item
 
 
+class PCMapItem(BaseModel):
```

Comment:
When are the scenarios when these are `None`?
Is there a scenario when they are `None`?
Should we add additional validation (perhaps full-model validation) where we make sure that at least one of these fields are set?

Reply:
@unparalleled-js My answer will be Vyper-specific for now: 

They can be none if the Vyper compiler doesn't have line information:

https://github.com/vyperlang/vyper/blob/b13595f4eb7472e298bb118a161f4e85035421cf/vyper/ir/compile_ir.py#L182

But I just realized the whole value could be `None` rather than a list of `None`s, so I need to fix that too:

https://github.com/vyperlang/vyper/blob/b13595f4eb7472e298bb118a161f4e85035421cf/vyper/ir/compile_ir.py#L745

Will be updating on Monday

Reply:
Updated in 4a9eb14